src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/executor/ExtractExecutorImpl.java

```diff
@@ -76,12 +77,50 @@ public ExtractExecutorImpl(
     return input;
   }
 
+  private static String getTeradataTimestampFromInstant(Instant instant) {
+    String instantWithoutNanoseconds =
+        instant.truncatedTo(ChronoUnit.SECONDS).toString().replaceAll("[TZ]", " ").trim();
+    // Convert nanosecond representation of Instant into fraction-of-second representation with a
+    // 6-digit max precision to conform with Teradata's TIMESTAMP format.
+    DecimalFormat formatter = new DecimalFormat();
+    formatter.setMinimumIntegerDigits(0);
+    formatter.setMaximumFractionDigits(6);
+    return instantWithoutNanoseconds + formatter.format(instant.getNano() / Math.pow(10, 9));
+  }
+
+  private static void maybeAddTimeRange(
+      SqlScriptVariables.QueryLogsVariables.Builder builder, Arguments arguments) {
+    boolean will_add = false;
```

Comment:
Variable names should use camelCase (i.e., use `willAdd` instead).

Also, the name is not very clear. How about `willAddTimeRange`.